app/Http/Controllers/ManageOrdersController.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Enum\DeliveryStatus;
+use App\Models\Order;
+use App\Models\OrderStatus;
+use Exception;
+use Illuminate\Http\Request;
+use Carbon\Carbon;
+
+class ManageOrdersController extends Controller
+{
+    public function index()
+    {
+        $orders = Order::query()->sortable()->paginate(10);
+
+        return view('admin.orders', [
+            'orders' => $orders,
+            'search' => null,
+            'dateFrom' => null,
+            'dateTill' => null,
+            'allstatusses' => $this->getAllStatusses(),
+            'selected' => null,
+        ]);
+    }
+
+    public function orderDetails(string $id)
+    {
+        $order = Order::find($id);
+        if ($order === null)
+        {
+            return redirect()->route('manage.orders.index')
+                ->with([
+                    'toast-type' => 'error',
+                    'toast-message' => __('manage-orders/order.order-doesnt-exist')
+                ]);
+        }
+
+        return view('admin.order-details', [
+            'order' => $order,
+        ]);
+    }
+
+    public function cancelOrder(string $id)
+    {
+        $order = Order::find($id);
+        if ($order === null)
+        {
+            return redirect()->route('manage.orders.index')
+                ->with([
+                    'toast-type' => 'error',
+                    'toast-message' => __('manage-orders/order.order-doesnt-exist')
+                ]);
+        }
+
+        $order->order_status_id = OrderStatus::where('status', DeliveryStatus::Cancelled)->first()->id;
```

Comment:
```suggestion
        $order->order_status_id = OrderStatus::where('status', DeliveryStatus::Cancelled->value)->first()->id;
```
Volgensmij werkt deze regel code anders niet

Reply:
Apart, had die wel aangepast, misschien in test branch